DTS-STN/Alpha-Site-CMS
# Set JWT expiry to 8 hours

# Description

[Set JWT expiry to 8h for Strapi admin sessions](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-593)

Small PR that changes the default JWT expiry time for admin sessions. Got rid of the extra config options to match what is currently in PR #31 

## Acceptance Criteria

Admin sessions timeout after each workday (8 hours) forcing the user to login again.

## Test Instructions

1. Pull in branch
2. Type `yarn develop`
3. Login to Strapi
4. Inspect page and go to Session Storage (or Local Storage if you clicked "Remember me"), under the Application tab (on Chrome)
5. Copy the value of `jwtToken`
6. Paste into the box on [https://jwt.io/](https://jwt.io/)
7. Hover your mouse over `iat` and `exp` to ensure that the expiry is set 8 hours from the issue date


## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
